dataplane/reconcilers_nonlinux.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !linux
+
+// Package dataplane is an implementation of the dataplane HAL API.
+package dataplane
+
+import (
+	"github.com/openconfig/lemming/dataplane/handlers"
+	"github.com/openconfig/lemming/dataplane/internal/engine"
+	"github.com/openconfig/lemming/gnmi/reconciler"
+)
+
+func getReconcilers(e *engine.Engine) []reconciler.Reconciler {
+	return []reconciler.Reconciler{handlers.NewRoute(e)}
```

Comment:
On a non-linux machine would this mean that the dataplane receives routes, programs tables, but doesn't actually do anything else (e.g. react to interface config and reflect their states). If so should there be another such build constraint for lemming.New such that we error out if dataplane is turned on? https://github.com/openconfig/lemming/commit/e0ca9054bc2ba84782a0f41725ec8ebad17b8000

Reply:
I've implemented this a runtime check, to reduce the number of _linux files we have